### ISSUE TITLE
connection problem

### ISSUE THREAD

**Issue by bonuSs311** [pedroSG94/RootEncoder#1326]
Please help me, what could be the problem? When I try to connect via rtmp it says the following:

FlashMainStream.cpp[69] Client RTMP not compatible with AMF3, few complex object can be not supported
FlashMainStream.cpp[150] Method client releaseStream not found in application live/
FlashMainStream.cpp[150] Method client FCPublish not found in application live/

**Comment by pedroSG94** [comment]
Hello,

Can you tell me rtmp server used?

**Comment by bonuSs311** [comment]
What I did was open OBS Studio, navigate to the stream section, and add my local IP to the RTMP server (rtmp://192.168.1.3/live). The stream key I used is "test". After that, I started MonaServer on my PC. I then opened your application, Root Encoder, on my mobile phone and selected "custom RTMP". I entered the address as follows: rtmp://192.168.1.3/live/test. However, an error message appeared on MonaServer, stating "Client RTMP not compatible with AMF3; few complex objects may not be supported
